docs/installation.rst

```diff
@@ -1,82 +1,113 @@
 Installation
 ============
 
+Built distributions
+-------------------
+
+Built distributions don't require to compile Shapely and its dependencies,
+and can be installed using ``pip`` or ``conda``. In addition, Shapely is also
+available via system package management tools like apt, yum, and Homebrew.
```

Comment:
Yeah, I copied this from the text in the README, but agreed that we should maybe not mention it (certainly if it is not even possible, like in the case of Homebrew).

> (installing GEOS via package manager then compiling Shapely using that may make sense, but as an advanced case)

I think that is indeed a use case. I certainly have seen issues of people install GEOS with brew, and then installing shapely on top of that. But then that doesn't belong in this section.

Reply:
Shapely 1.8 seems to be available on debian unstable (https://packages.debian.org/search?keywords=python3-shapely), so that seems relatively up to date. 
So maybe I will generalize the sentence a bit, and remove the reference of Homebrew.